### PR TITLE
Use actionproxy from dockerSkeleton do not clown and own is the motto FTW!

### LINKED ISSUE #3: python 3 directory not properly named
we should follow a standard convention and best practice for naming the runtimes.

### PR DESCRIPTION
**Pull request by csantanapr**
Closes #3

### CODE REVIEW

**Comment by rabbah** [comment]
No more clowning around!

File: ansible/environments/local/group_vars/all [hunk 1]
```diff
@@ -19,11 +20,19 @@ apigw_auth_user: ""
...
 invoker_allow_multiple_instances: true
 
+# Set kafka configuration
+kafka_heap: '512m'
+kafka_topics_completed_retentionBytes: 104857600
+kafka_topics_completed_retentionMS: 300000
+kafka_topics_health_retentionBytes: 104857600
+kafka_topics_health_retentionMS: 300000
+kafka_topics_invoker_retentionBytes: 104857600
+kafka_topics_invoker_retentionMS: 300000
```
Comment (rabbah) [inline ansible/environments/local/group_vars/all]: hmm you shouldn't need these

Comment (csantanapr) [reply ansible/environments/local/group_vars/all]: hmmm then why do we have them here then?
https://github.com/apache/incubator-openwhisk/blob/master/ansible/environments/local/group_vars/all#L27-L34

**Comment by csantanapr** [comment]
@rabbah bump

**Pull request merged**